packages/gatsby-core-utils/package.json

```diff
@@ -50,7 +50,7 @@
     "fs-extra": "^10.1.0",
     "got": "^11.8.3",
     "import-from": "^4.0.0",
-    "lmdb": "2.3.10",
+    "lmdb": "2.4.2",
```

Comment:
Typo?

Reply:
what do you mean with typo? I can't see it

Reply:
lol, scratch that. I read 2.3.2 🤣